Add alert on disabling metric collection (#25509)
DataDog/documentation: content/en/logs/guide/how-to-set-up-only-logs.md

  - /logs/faq/how-to-set-up-only-logs
---

> <div class="alert alert-danger">Infrastructure Monitoring is a prerequisite to using APM. If you are an APM customer, do not turn off metric collection so that you don't lose critical telemetry and value.</div>

Reviewer: @estherk15 what do you mean by `value` here? 

How about this:

> If you are an APM customer, do not turn off metric collection or you might lose critical telemetry and metric collection information.

> <div class="alert alert-danger">Infrastructure Monitoring is a prerequisite to using APM. If you are an APM customer, do not turn off metric collection so that you don't lose critical telemetry and value.</div>

Reviewer: ```suggestion
<div class="alert alert-danger">Infrastructure Monitoring is a prerequisite to using APM. If you are an APM customer, do not turn off metric collection or you might lose critical telemetry and metric collection information.</div>
```
